packages/record-data/addon/-private/normalize-link.ts

```diff
@@ -1,19 +1,18 @@
-/**
-  @module @ember-data/store
-*/
-
 /*
   This method normalizes a link to an "links object". If the passed link is
   already an object it's returned without any modifications.
 
   See http://jsonapi.org/format/#document-links for more information.
 
   @method _normalizeLink
-  @private
+  @internal
```

Comment:
Is this an actual valid pragma?

Reply:
yes, it prevents it from showing in the docs at all which is what we want given this method has never been user importable